src/ResourceManager/Sql/Commands.Sql/Replication/Model/AzureSqlDatabaseCopyModel.cs

```diff
@@ -55,5 +56,10 @@ public class AzureSqlDatabaseCopyModel : AzureSqlDatabaseReplicationModelBase
         /// Gets or sets the location of the partner database
         /// </summary>
         public string CopyLocation { get; set; }
+
+        /// <summary>
+        /// Gets or sets the requested Sku for the copy
+        /// </summary>
+        public Sku Sku { get; set; }
```

Comment:
The design here should be the same as AzureSqlDatabaseModel. Can you flatten this out?

Reply:
It is not displayed because of filter in Types.ps1xml file. However I think that we are still not supposed to directly expose the SDK types.

Reply:
@jaredmoo Remove the Sku in CopyModel and SecondarySku in ReplicationLinkModel to avoid exposing SDK types in powershell.